src/registry.cpp

```diff
@@ -36,6 +36,7 @@
 #endif
 
 std::string Registry::ReadStrValue(HKEY hkey, std::string const& key, std::string const& val, REGVIEW view) {
+#ifndef _ARM_
```

Comment:
The ARM toolchain doesn't export symbols for this. Will improve the stubbing to match other port behaviour.